src/api/article-api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+const newsAPI = axios.create({
+    baseURL: "https://news-app-4jdh.onrender.com/api/"
+})
+
+
+export const getArticles = async (articleID = null) => {
```

Comment:
This api request could get a bit bloated once we start adding queries, you may want separate requests for all articles and a single article as they are different api calls.